### PR TITLE
Changed/Fixed Lambda to anonymous class of ElementPolicy in doc

### PR DESCRIPTION
**Pull request by 0xflotus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,7 +67,8 @@ PolicyFactory policy = new HtmlPolicyBuilder()
     .allowElements("p")
     .allowElements(
         new ElementPolicy() {
-          (String elementName, List<String> attrs) -> {
+          @Override
+          public String apply(String elementName, List<String> attrs) {
```
Comment (mikesamuel) [inline README.md]: Thanks for catching that.   Would it be sufficient to just get rid of the `new ElementPolicy` stuff and leave the lambda invocation?

Comment (0xflotus) [reply README.md]: Sure.

**Comment by mikesamuel** [comment]
Thanks much.

File: README.md [hunk 2]
```diff
@@ -67,12 +67,11 @@ PolicyFactory policy = new HtmlPolicyBuilder()
     .allowElements("p")
     .allowElements(
         (String elementName, List<String> attrs) -> {
-            // Add a class attribute.
-            attrs.add("class");
-            attrs.add("header-" + elementName);
-            // Return elementName to include, null to drop.
-            return "div";
-          }
```
Comment (0xflotus) [inline README.md]: oops. Sorry, didn't recognized it.

**Pull request merged**